testsuite/run.py

```diff
@@ -30,6 +30,16 @@ def set_up(self):
         # Some tests rely on an initially empty GPR_PROJECT_PATH variable
         os.environ.pop('GPR_PROJECT_PATH', None)
 
+        # Let's run `alr` from project's build dir
+        alr_path = self._get_alr_path()
+        self._prepend_alr_to_path(alr_path)
+
+    def _get_alr_path(self):
```

Comment:
Should we add these functions (`_get_alr_path` and `_prepend_alr_to_path`) here, or in `PythonScriptDriver`?